pkgs/tools/security/pyrit/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchFromGitHub, python2Packages, openssl, zlib, libpcap, opencl-headers, ocl-icd }:
```

Comment:
could we use python3 instead of python2?

Reply:
Unfortunately, not yet. The message at the top of the README here https://github.com/JPaulMora/Pyrit indicates that they're working on it.